pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -163,6 +163,11 @@ func init() {
 			rbac.NewRule("create").Groups(kAuthzGroup).Resources("subjectaccessreviews").RuleOrDie(),
 			rbac.NewRule("update").Groups(templateGroup).Resources("templateinstances/status").RuleOrDie(),
 			rbac.NewRule("update").Groups(templateGroup).Resources("templateinstances/finalizers").RuleOrDie(),
+			// template instance controller needs to be able to create and get any resource the user is allowed to create.
+			rbac.NewRule("create").Groups(rbac.APIGroupAll).Resources(rbac.ResourceAll).RuleOrDie(),
+			rbac.NewRule("get").Groups(rbac.APIGroupAll).Resources(rbac.ResourceAll).RuleOrDie(),
+			// delete needed to be able to set ownerrefs on objects
+			rbac.NewRule("delete").Groups(rbac.APIGroupAll).Resources(rbac.ResourceAll).RuleOrDie(),
```

Comment:
@smarterclayton @liggitt @deads2k let the debate begin...

Reply:
For blockOwnerDeletion? I thought it was update of `*/finalizers`

Reply:
That makes sense on an update, I didn't realize that was required to set an ownerRef on create. Is that what you're seeing?

Reply:
I believe so but I didn't look at it super closely. (given that I didn't grant the controller update permissions, though, it seems like it would have to have been on create)